scripts/build

```diff
@@ -11,31 +11,33 @@ cd "$SCRIPTS_DIR/.."
 #########
 
 function title() {
-    echo "Build Internet Identity"
+    echo "Builds Internet Identity and Archive Canisters"
 }
 
 function usage() {
     cat << EOF
 
 Usage:
-  $0 [--only-dependencies]
+  $0 [--only-dependencies] [--single-canister CANISTER]
```

Comment:
nit: how about `--internet-identity` to enable the II build, and `--archive` to enable the archive build? shorter to type and more flexible; would default to `--internet-identity --archive`. Would also simplify the logic around `$CANISTER`